mobile/Ihc/screens/newPatientHomeScreen.js

```diff
@@ -0,0 +1,180 @@
+import React, { Component } from 'react';
+import {
+  AppRegistry,
+  StyleSheet,
+  Text,
+  View,
+  Modal,
+  TouchableHighlight,
+  TouchableOpacity,
+  Animated
+} from 'react-native';
+import ScrollableTabView from 'react-native-scrollable-tab-view';
+import TabBar from "react-native-underline-tabbar";
+import Button from '../components/Button';
+import TriageScreen from './TriageScreen';
+import TriagePatientPage from '../components/TriagePatientPage';
+import Table from 'react-native-simple-table';
+import PatientHomeScreen from './PatientHomeScreen';
+
+const Page = ({label}) => (
+    <View style={styles.container}>
+      <Text style={styles.welcome}>
+        {label}
+      </Text>
+      <Text style={styles.instructions}>
+        To get started, edit index.ios.js
```

Comment:
These instructional texts don't look relevant for the public facing app... if it is just there as a placeholder, then leave a TODO